### DISCUSSION TITLE
How to update please?

### DISCUSSION THREAD

**Discussion by B17Fan** [CorsixTH/CorsixTH#2298] [General Diagnostics] [answered]
I've think I installed CorsixTH in 2020. I want to update it, but I've got no idea what files I'm supposed to put where. Can someone please tell me, step by step, how to update my game please? Many thanks

**Comment by ravenPenfold** [comment, marked as answer]
Hi.

### Windows

If you hop over to [Getting Started wiki page](https://github.com/CorsixTH/CorsixTH/wiki/Getting-Started), and follow the instructions as if you were installing, the installer should either detect an older version, and ask if you want to update, or state that you will be overwriting files (see dialog image below).
![image](https://user-images.githubusercontent.com/110283510/211058234-1e81211b-1331-4e46-be0d-4ebb2bd1e78b.png)
You should click on 'Yes' in this dialog, and it should update it to the latest version.

### Debian Based Linux Distros (Ubuntu, etc)
If you either use your package manager, or open a terminal emulator and type in ``apt install corsixth``

### Any others
Follow the installation instructions at [Getting Started wiki page](https://github.com/CorsixTH/CorsixTH/wiki/Getting-Started), and you should be good to go.
